kasa/aestransport.py

```diff
@@ -184,8 +185,24 @@ async def send_secure_passthrough(self, request: str) -> Dict[str, Any]:
             assert self._encryption_session is not None
 
         raw_response: str = resp_dict["result"]["response"]
-        response = self._encryption_session.decrypt(raw_response.encode())
-        return json_loads(response)  # type: ignore[return-value]
+
+        try:
+            response = self._encryption_session.decrypt(raw_response.encode())
+            ret_val = json_loads(response)
+        except Exception as ex:
```

Comment:
Can we limit down the exceptions here to just the ones that it would throw on bad encryption

Reply:
I wasn't sure how to get a full set because I was getting a few different exceptions depending on the actual response, padding, block sizes etc.  I don't see this having too much impact in practice because this it the only instance when I've seen the decrypt fail, i.e. when it not encrypted at all.

Reply:
Mostly, I'm worried we're going to create hard to find bugs cause we're swallowing too many exceptions, but we're already doing that below anyway so I guess it doesn't matter so much